.pre-commit-config.yaml

```diff
@@ -18,16 +18,9 @@ repos:
     rev: 1.4.0
     hooks:
     - id: php-lint-all
-    - id: php-cs
```

Comment:
if removal of this because it would duplicate psalm?

Reply:
phpcbf actually uses php-cs as the underlying scanner. So in effect, we're doing the same scan twice.

Reply:
I removed phpstan as it seems pointless to have two linters doing the same job, which may in fact contradict each other. phpcbf does a slightly different job, as it enforces code style, rather than code correctness.